posthog-node/test/posthog-node.spec.ts

```diff
@@ -381,33 +381,29 @@ describe('PostHog Node.js', () => {
       })
       ph.debug(true)
 
-      // using debug mode to check console.log output
-      // which tells us when the flush is complete
-
-      ph.capture({ event: 'test-event', distinctId: '123' })
-      await wait(100)
-      expect(logSpy).toHaveBeenCalledTimes(1)
-
-      ph.capture({ event: 'test-event', distinctId: '123' })
-      ph.capture({ event: 'test-event', distinctId: '123' })
-      await wait(100)
-      expect(logSpy).toHaveBeenCalledTimes(3)
-      await wait(400) // The flush will resolve in this time
-      ph.capture({ event: 'test-event', distinctId: '123' })
-      ph.capture({ event: 'test-event', distinctId: '123' })
-      await wait(100)
-      expect(logSpy).toHaveBeenCalledTimes(6) // 5 captures and 1 flush
-      expect(5).toEqual(logSpy.mock.calls.filter((call) => call[1].includes('capture')).length)
-      expect(1).toEqual(logSpy.mock.calls.filter((call) => call[1].includes('flush')).length)
-
-      logSpy.mockClear()
-      expect(logSpy).toHaveBeenCalledTimes(0)
-
-      console.warn('YOO!!')
+      ph.capture({ event: 'test-event-1', distinctId: '123' })
+
+      // start flushing, but don't wait for promise to resolve before resuming events
+      void ph.flush()
```

Comment:
Could store this in a var and assert it was resolved by `shutdown`

Reply:
Done - this felt pretty hacky but it's test code so at least if it breaks in a version update, it'll only affect us and not users